frontend/src/hooks/useCopyToClipboard.ts

```diff
@@ -0,0 +1,27 @@
+"use client";
```

Comment:
These two hooks are user driven, so not using `useEffect()` even though they touch outside systems.